test/models/changelog_entry_tweet_formatter_test.rb

```diff
@@ -0,0 +1,40 @@
+require "test_helper"
+
+class ChangelogEntryTweetFormatterTest < ActiveSupport::TestCase
+  test "returns empty string if tweet_copy is empty" do
+    entry = create(:changelog_entry, tweet_copy: nil)
+
+    assert_equal "", ChangelogEntryTweetFormatter.format(entry)
```

Comment:
Should this return if the tweet_copy is empty, to help ensure we never tweet it empty?